quickstart/scaling_etcd.md

```diff
@@ -0,0 +1,230 @@
+
+
+{% tabs %}
+{% tab title="Command" %}
+```text 
+kubectl edit sts mayastor-etcd -n mayastor 
+```
+{% endtab %}
+{% tab title="Output" %}
+```text 
+        - name: ETCD_INITIAL_CLUSTER_STATE
+          value: existing
+        - name: ETCD_INITIAL_CLUSTER
+          value: mayastor-etcd-0=http://mayastor-etcd-0.mayastor-etcd-headless.mayastor.svc.cluster.local:2380,mayastor-etcd-1=http://mayastor-etcd-1.mayastor-etcd-headless.mayastor.svc.cluster.local:2380,mayastor-etcd-2=http://mayastor-etcd-2.mayastor-etcd-headless.mayastor.svc.cluster.local:2380,mayastor-etcd-3=http://mayastor-etcd-3.mayastor-etcd-headless.mayastor.svc.cluster.local:2380
+```
+{% endtab %}
+{% endtabs %}
+
+
+## Step 3: Create a Persistent Volume
+
+Create a PV with the following YAML. Change the pod name/claim name based on the pod's unique identity.
+
+
+{% tabs %}
+{% tab title="YAML" %}
+```text 
+
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    meta.helm.sh/release-name: mayastor
+    meta.helm.sh/release-namespace: mayastor
+    pv.kubernetes.io/bound-by-controller: "yes"
+  finalizers:
+  - kubernetes.io/pv-protection
+  labels:
+    app.kubernetes.io/managed-by: Helm
+    statefulset.kubernetes.io/pod-name: mayastor-etcd-3
+  name: etcd-volume-3
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 2Gi
+  claimRef:
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    name: data-mayastor-etcd-3
+    namespace: mayastor
+  hostPath:
+    path: /var/local/mayastor/etcd/pod-3
+    type: ""
+  persistentVolumeReclaimPolicy: Delete
+  storageClassName: manual
+  volumeMode: Filesystem
+```
+{% endtab %}
+{% tab title="Output" %}
+```text 
+kubectl apply -f pv-etcd.yaml -n mayastor
+persistentvolume/etcd-volume-3 created
+```
+{% endtab %}
+{% endtabs %}
+
+
+
+
+## Step 4: Validate Key-Value Pairs
+
+
+Run the following command from the new ETCD pod and ensure that the values are the same as those in etcd-0/1/2. Otherwise, it indicates a data loss issue.
+
+
+{% tabs %}
+{% tab title="Command" %}
+```text 
+root@543-master:~# kubectl exec -it mayastor-etcd-3 -n mayastor -- bash
+Defaulted container "etcd" out of: etcd, volume-permissions (init)
+I have no name!@mayastor-etcd-3:/opt/bitnami/etcd$ ETCDCTL_API=3
+I have no name!@mayastor-etcd-3:/opt/bitnami/etcd$ etcdctl get --prefix ""
+```
+{% endtab %}
+{% tab title="Output" %}
+```text 
+/openebs.io/mayastor/apis/v0/clusters/ce05eb25-50cc-400a-a57f-37e6a5ed9bef/namespaces/mayastor/CoreRegistryConfig/db98f8bb-4afc-45d0-85b9-24c99cc443f2
```

Comment:
Again, should we be printing the entire contents here?

Reply:
We can remove them and put just the command alone.